jme3-core/src/main/java/com/jme3/light/SpotLight.java

```diff
@@ -74,6 +75,39 @@ public SpotLight() {
         computeAngleParameters();
     }
 
+    /**
+     *
+     * @param lightColor Color of the light
+     * @param position Position of the light
+     * @param direction Direction of the light
+     * @param spotInnerAngle Inner angle of the light
+     * @param spotOuterAngle Outer angle of the light
+     */
+    public SpotLight(ColorRGBA lightColor,Vector3f position, Vector3f direction, float spotInnerAngle, float spotOuterAngle) {
+        this();
+        setColor(lightColor);
+        setPosition(position);
+        setDirection(direction);
+        setSpotInnerAngle(spotInnerAngle);
+        setSpotOuterAngle(spotOuterAngle);
+        computeAngleParameters();
+    }
+
+    /**
+     *
+     * @param lightColor Color of the light
+     * @param position Position of the light
+     * @param direction Direction of the light
+     * @param spotInnerAngle Inner angle of the light
+     * @param spotOuterAngle Outer angle of the light
+     * @param spotRange Range of the light
+     */
+    public SpotLight(ColorRGBA lightColor,Vector3f position, Vector3f direction, float spotInnerAngle, float spotOuterAngle,float spotRange) {
+        this(lightColor,position,direction,spotInnerAngle,spotOuterAngle);
+        setSpotRange(spotRange);
+        computeAngleParameters();
```

Comment:
This call to computeAngleParameters(); can be removed as it's been called in the prior thi(...) call